lib/gssapi/mech/gss_mech_switch.c

```diff
@@ -167,7 +167,7 @@ do {									\
 	m->gm_mech.gm_ ## name = (_gss_##name##_t *)dlsym(so, "gss_" #name); \
 	if (!m->gm_mech.gm_ ## name ||					\
 	    m->gm_mech.gm_ ##name == gss_ ## name) {			\
-		_gss_mg_log(1, "can't find symbol gss_" #name "\n");	\
+		_gss_mg_log(1, "can't find symbol gss_" #name);		\
```

Comment:
Do we want to log the name of the DSO we're looking in here?

Reply:
Here, I was just eliminating the "\n" because it puts extra blank lines in. It's a separate commit from the other stuff.